src/sources/opentelemetry/tests.rs

```diff
@@ -26,21 +27,32 @@ fn generate_config() {
 }
 
 #[tokio::test]
-async fn receive_message() {
+async fn receive_grpc_logs() {
```

Comment:
Would it make sense to have a similar test for the HTTP endpoint that makes similar assertions?

Reply:
Honestly, I'm not even sure this existing test is super valuable. It's more testing the `opentelemetry::convert` code than the server. Aside from exercising `convert` we're also just testing `tonic` as well, which IMHO isn't super valuable.

Maybe I'm leaning into guarantees/trust on the protobuf model here, but as mentioned in a reply to @neuronull below - the HTTP side of things could probably do some assertions that our responses match the expectations of the spec since that's really the only code that we've "written" here.

Reply:
Going to continue to discuss with the team offline.